.github/workflows/ci.yml

```diff
@@ -27,7 +27,6 @@ jobs:
           - stack-lts-18.28.yaml # ghc-8.10
           - stack-lts-16.31.yaml # ghc-8.8
           - stack-lts-14.27.yaml # ghc-8.6
-          - stack-lts-12.26.yaml # ghc-8.4
```

Comment:
I don't think this is controversial.

Reply:
:+1: `freckle-app` in particular shouldn't come with high compatibility requirements. It's only when we're talking about a library of ours that is a deps-of-deps situation that dropping support can be a cascading annoyance.